deploy/standard/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -16,8 +16,9 @@ metadata:
             "namespace":"kubevirt-hyperconverged"
           },
           "spec": {
-            "CDIImagePullPolicy":"IfNotPresent",
-            "KubeVirtImagePullPolicy":"IfNotPresent"
+            "Version":"2.0",
```

Comment:
We may need to remove the deploy directory entirely to prevent people from modifying generated files...or remove the generator. You must modify the template `templates/olm-catalog/kubevirt-hyperconverged/VERSION/kubevirt-hyperconverged-operator.VERSION.clusterserviceversion.yaml.in` for this to not get blown away by the generator.

Reply:
ah thanks, I went to the wrong dir.